pkg/webui/components/tabs/tab/index.js

```diff
@@ -81,13 +81,18 @@ class Tab extends React.PureComponent {
       [style.tabItemDisabled]: disabled,
     })
 
-    const Component = link ? NavLink : 'span'
+    // There is no support for disabled on anchors in html and hence in `react-router`. So, do
+    // not render the link component if the tab is disabled, but render regular tab item instead.
+    const canRenderLink = link && !disabled
+
```

Comment:
Did you try putting the `disabled` prop on the link component? There is a `pointer-events: none` in its css that should prevent navigation.

Reply:
`NavLink` doesnt have the `disabled` prop

Reply:
Also, `pointer-events` property only disables mouse events which means that it is still possible to navigate through disabled links using the keyboard.